### ISSUE TITLE
Feature required - Custom update widget

### ISSUE THREAD

**Issue by devpankajpatel** [larryaasen/upgrader#303]
![image](https://github.com/larryaasen/upgrader/assets/110975475/8c4a0e9b-2621-4e7d-9cb9-1a13297c3ffb)

- You can use a function called checkUpdateAvailable() which will tell you whether an update is available or not. It will return true if there is an update available, and false if there isn't.
- Alternatively, you can use a function called displayToast() to directly show a toast message similar to the image you provided. You just need to pass the appropriate content to the function, and it will display the toast for you.

![image](https://github.com/larryaasen/upgrader/assets/110975475/f3ad2e00-266a-44ca-9cf7-1bd5b5802ed2)

- Allow users to personalize widgets according to their preferences, such as choosing whether to display the app logo or the play store icon. Additionally, provide options to customize the widget's appearance, such as selecting between a bottom sheet or a customizable dialog. This allows users to have full control over how the widget appears, just like the image provided.

**Comment by lisovyk** [comment]
yeah. highly needed aswell. Upgrader content is barely customizable, and that's the reason I am moving away from the production version right now.
